audinterface/core/process_with_context.py

```diff
@@ -232,36 +241,30 @@ def _process_signal_from_index(
             idx: int = 0,
             root: str = None,
             file: str = None,
-    ) -> pd.Series:
-
-        utils.assert_index(index)
+    ) -> typing.Any:
 
-        if len(index) == 0:
-            y = pd.Series([], index=index, dtype=object)
-        else:
-            starts_i, ends_i = utils.segments_to_indices(
-                signal,
-                sampling_rate,
-                index,
-            )
-            y = self._call(
-                signal,
-                sampling_rate,
-                starts_i,
-                ends_i,
-                idx=idx,
-                root=root,
-                file=file,
+        starts_i, ends_i = utils.segments_to_indices(
+            signal,
+            sampling_rate,
+            index,
+        )
+        y = self._call(
```

Comment:
Is `y` a series here?

Reply:
It's whatever your processing function returns. Most likely not a series.

Reply:
It would then be safer to not use `y` as variable name as usually you would assume it represents a series.
So maybe change the variable name to `results`?

Reply:
Mhh, also in many other places we assign the result of the processing function to `y`. If you think we should change that I propose we do it in a different PR.

Reply:
You are right, I created https://github.com/audeering/audinterface/issues/105 to track this.